### ISSUE TITLE
Test-suite: while vs for

### ISSUE THREAD

**Issue by wmvndrscht** [LangProc/langproc-2017-cw#5]
In the Test suit required tests:

`5 - **FOR**_N : Check that a **while** loop can execute more than once.`

Should this be:

`1. 5 - WHILE_N : Check that a while loop can execute more than once.`
or
`2. 5 - FOR_N : Check that a for loop can execute a loop with more than iteration.`

**Comment by m8pple** [comment]
Yes, you're right. I changed the test-cases slightly to discourage temporal translation
of submissions, and must have missed that.

Thanks, I'll update it.

**Comment by edpickup** [comment]
Could this be clarified in the spec? Its still the original wording

**Comment by edpickup** [comment]
Sorry to bump this again, but the spec still has the original wording, and it is unclear which of the two (while vs for) is correct.

**Comment by m8pple** [comment]
Given I completely failed to follow up on this, I'll simply transfer the problem to me.
I'll manually inspect each one, and work out whether it tests for a while loop or a
for loop.
